pkgs/development/compilers/llvm/15/clang/purity.patch

```diff
@@ -11,13 +11,12 @@ diff --git a/lib/Driver/ToolChains/Gnu.cpp b/lib/Driver/ToolChains/Gnu.cpp
 index fe3c0191bb..c6a482bece 100644
 --- a/lib/Driver/ToolChains/Gnu.cpp
 +++ b/lib/Driver/ToolChains/Gnu.cpp
```

Comment:
Why does this patch for LLVM 15 change when removing LLVM 14?

Reply:
More misunderstanding of the semantics of `getVersionFile` on my part. Will fix.